javascript/app-discover/src/components/GraphFilteringControls.tsx

```diff
@@ -26,28 +26,32 @@ export const GraphFilteringControls = memo(function GraphFilteringControls() {
 
 	return (
 		<GraphFilteringContainer>
-			<ThresholdSlider
-				label={'Edge weight threshold'}
-				thresholdState={WeightThresholdState}
-				defaultStyling
-			/>
-			<ThresholdSlider
-				label={'Edge confidence threshold'}
-				thresholdState={ConfidenceThresholdState}
-				defaultStyling
-			/>
 			<ThresholdSlider
 				label={'Correlation visibility threshold'}
 				thresholdState={CorrelationThresholdState}
 				defaultStyling
 			/>
-			{selectedCausalDiscoveryAlgorithm === CausalDiscoveryAlgorithm.DECI && (
-				<Toggle
-					label="Fixed intervention ranges"
-					checked={fixedInterventionRangesEnabled}
-					onChange={(e, v) => setFixedInterventionRangesEnabled(Boolean(v))}
+			{selectedCausalDiscoveryAlgorithm !== CausalDiscoveryAlgorithm.PC && (
```

Comment:
We could perhaps use react-if with `<When>` where.

Reply:
Sure! Since it's not installed on this app, we can do it in the refactor-phase since there's a lot of places that it could be used too